shared/settings/account/index.js

```diff
@@ -0,0 +1,120 @@
+// @flow
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import EmailPhoneRow from './email-phone-row'
+
+type Props = {|
+  contactKeys: Array<string>,
+  hasPassword: boolean,
+  onAddEmail: () => void,
+  onAddPhone: () => void,
+  onDeleteAccount: () => void,
+  onSetPassword: () => void,
+|}
+
+const AccountSettings = (props: Props) => {
+  let passwordLabel = props.hasPassword ? 'Change password' : 'Set a password'
+  if (props.hasPassword && Styles.isMobile) {
+    passwordLabel = 'Change'
+  }
+  return (
+    <Kb.ScrollView>
+      <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true}>
+        {/* Email + Phone */}
```

Comment:
if its worth commenting then i think its worth breaking out into its own component in this file